main.go

```diff
@@ -258,6 +304,65 @@ func listen(started chan bool, context *Context) {
 	context.listeners.Done()
 }
 
+// Open listening socket in client mode.
+func clientListen(started chan bool, reload_client chan bool, context *Context) {
+	// Serve /_status.
+	// reload_status is a channel which causes /_status to reload.
+	var reload_status chan bool
+	if *statusAddr != nil {
+		tlsConfigStatus, err := buildConfig(*keystorePath, *keystorePass, *caBundlePath, *tlsVersion)
+		if err != nil {
+			logger.Printf("failed to load tls config")
+			started <- false
+			return
+		}
+		tlsConfigStatus.ClientAuth = tls.NoClientCert
+		statusListener := serveStatus(tlsConfigStatus, context)
+
+		reload_status = make(chan bool, 1)
+		go func() {
```

Comment:
Correct. In server mode, we handle reloading by calling serverListen multiple times. In client mode, we handle reloading inside clientListen.